jdaviz/configs/cubeviz/helper.py

```diff
@@ -79,6 +80,28 @@ def load_data(self, data, data_label=None, override_cube_limit=False, **kwargs):
 
         super().load_data(data, parser_reference="cubeviz-data-parser", **kwargs)
 
+        if 'Spectral Extraction' not in self.plugins.keys():  # pragma: nocov
+            msg = SnackbarMessage(
+                "Automatic spectral extraction requires the Spectral Extraction plugin to be enabled",  # noqa
+                color='error', sender=self, timeout=10000)
+            self.app.hub.broadcast(msg)
+        else:
+            try:
+                self.plugins['Spectral Extraction']._obj._extract_in_new_instance(auto_update=False, add_data=True)  # noqa
+            except Exception:
+                msg = SnackbarMessage(
+                    "Automatic spectrum extraction for the entire cube failed."
+                    " See the spectral extraction plugin to perform a custom extraction",
+                    color='error', sender=self, timeout=10000)
+                raise  # TODO: remove before merge
```

Comment:
```suggestion
```

Reply:
thanks for the reminder - now that tests are passing, its FINALLY time to remove this 😂